static/openapi/vwrs-spec.json

```diff
@@ -2109,13 +1629,13 @@
                         "domain_key": "test.domain",
                         "requests": 1305,
                         "max_rps": 100,
-                        "timestamp": "2023-07-12T07:00:00Z"
+                        "timestamp": "2023-07-12T07:00:00.000Z"
```

Comment:
Are you sure about the subsecond resolution?  Do we need that kind of resolution?

Reply:
Oddly enough, `swagger.yaml` doesn't have this. I guess it's generated because of some change to `editor.swagger.io`. With or without it, the ISO-8601 timestamp is still valid, so I may as well remove it.